packages/venia-concept/package.json

```diff
@@ -13,7 +13,7 @@
     "venia": "./bin/venia.js"
   },
   "scripts": {
-    "build": "yarn run clean && yarn run build:esm && yarn run validate-queries && yarn run build:prod",
+    "build": "yarn run clean && yarn run build:esm && yarn run build:prod",
```

Comment:
Por que no mas `validate-queries`?

Reply:
I don't want to delete this in my real PR but it unblocked me while my magento backend certs are expired. waiting on adobe to issue new certs so we can spin those back up. all the regression environments that folks test magento releases with are all turned off at the moment for this and this is where our backends live as well.

Reply:
tldr; the build fails if the validation of queries fails, which happens when the cert is expired